src/encord_active/cli/utils/encord.py

```diff
@@ -62,12 +85,16 @@ def import_encord_project(ssh_key_path: Path, target: Path, encord_project_hash:
     project_path.mkdir(exist_ok=True, parents=True)
     project_file_structure = ProjectFileStructure(project_path)
 
+    ontology = OntologyStructure.from_dict(project.ontology)
+    option_hashes_to_tag = suggest_tagging_data(ontology)
+
     meta_data = {
         "project_title": project.title,
         "project_description": project.description,
         "project_hash": project.project_hash,
         "ssh_key_path": ssh_key_path.as_posix(),
         "has_remote": True,
+        "nested_attributes_to_tag": list(option_hashes_to_tag),
```

Comment:
[curious]🤔 

Why do we need this as part of the project meta? (we don't seem to use it afterwards)